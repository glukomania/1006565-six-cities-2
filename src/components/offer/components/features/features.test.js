import renderer from 'react-test-renderer';
import Features from './features';

it(`Features of property are displayed correctly`, () => {
  const features = renderer
    .create(<Features
      entire={`Entire`}
      bedroom={0}
      adults={0}
    />)
  .toJSON();
  expect(features).toMatchSnapshot();
});
