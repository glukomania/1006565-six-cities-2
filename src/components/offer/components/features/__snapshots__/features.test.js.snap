// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features of property are displayed correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Entire place
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    0
     bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    0
     adults
  </li>
</ul>
`;
